src/test/suite/index.ts

```diff
@@ -36,15 +36,14 @@ function loadCoverageRunner(testsRoot: string): CoverageRunner | undefined {
 	}
 }
 
-let mocha = new Mocha({
+// Create the mocha test
+const mocha = new Mocha({
 	ui: 'tdd',
 	timeout: 20000,
 	reporter: 'mocha-jenkins-reporter',
-	useColors: true
```

Comment:
regarding https://github.com/mochajs/mocha/pull/4178 we can still use `colors` instead of `useColors`.

Reply:
good to know. I wasn't sure what to do with it, but useColors definitely threw an error

Reply:
looks like it's color instead of colors, but I'm trying it here

Reply:
yup. that worked.